src/App.js

```diff
@@ -8,6 +8,7 @@ import Playlist from "./pages/Playlist/Playlist";
 import Liked from "./pages/Liked/Liked";
 import WatchLater from "./pages/WatchLater/WatchLater";
 import History from "./pages/History/History";
+import Profile from "./pages/Profile/Profile";
```

Comment:
You can make a utility file and import these components together, for cleaner code.